sdk/react/src/lib/fragment-components/a-date-picker.tsx

```diff
@@ -0,0 +1,364 @@
+						? `<ibm-date-picker
+							${angularClassNamesFromComponentObj(json)}
+							${json.rangeStartLabel ? `[label]='${json.rangeStartLabel}'` : ''}
+							${json.light ? '[light]="true"' : ''}
+							${json.disabled ? `[disabled]='${json.disabled}'` : ''}
+							${json.invalid ? `[invalid]='${json.invalid}'` : ''}
+							${json.invalidText ? `[invalidText]='${json.invalidText}'` : ''}
+							[placeholder]="${json.placeholder}"
+							[size]="${json.size}"
+							[value]="${nameStringToVariableString(json.codeContext?.name)}Value"
+							[dateFormat]="${json.dateFormat}"
+							(valueChange)="${nameStringToVariableString(json.codeContext?.name)}ValueChange.emit($event.value)">
+						</ibm-date-picker>`
+						: `<ibm-date-picker
+							${angularClassNamesFromComponentObj(json)}
+							${json.rangeStartLabel ? `[label]='${json.rangeStartLabel}'` : ''}
+							${json.rangeEndLabel ? `[rangeLabel]='${json.rangeEndLabel}'` : ''}
+							${json.light ? '[light]="true"' : ''}
+							${json.disabled ? `[disabled]='${json.disabled}'` : ''}
+							${json.invalid ? `[invalid]='${json.invalid}'` : ''}
+							${json.invalidText ? `[invalidText]='${json.invalidText}'` : ''}
+							[size]="${json.size}"
+							range="true"
+							[placeholder]="${json.placeholder}"
+							[dateFormat]="${json.dateFormat}"
+							[value]="${nameStringToVariableString(json.codeContext?.name)}Value"
+							(valueChange)="${nameStringToVariableString(json.codeContext?.name)}ValueChange.emit($event.value)">
+						</ibm-date-picker>`
+					}`;
+				}
+			}
+		},
+		react: {
+			latest: {
+				imports: ['DatePicker', 'DatePickerInput'],
+				code: ({ json }) => {
+					return `<DatePicker
+								${reactClassNamesFromComponentObj(json)}
+								dateFormat="${json.dateFormat}"
+								datePickerType="${json.datePickerType}"
+								${json.light ? 'light="true"' : ''}>
+								<DatePickerInput
+									placeholder="${json.placeholder}"
+									${json.rangeStartLabel ? `labelText='${json.rangeStartLabel}'` : ''}
+									${json.disabled ? `disabled='${json.disabled}'` : ''}
+									${json.invalid ? `invalid='${json.invalid}'` : ''}
+									${json.invalidText ? `invalidText='${json.invalidText}'` : ''}
+									size="${json.size}"
+									type='text'
+								/>
+								${json.datePickerType === 'range'
+									? `<DatePickerInput
+										${json.rangeEndLabel ? `labelText='${json.rangeEndLabel}'` : ''}
+										${json.disabled ? `disabled='${json.disabled}'` : ''}
+										${json.invalid ? `invalid='${json.invalid}'` : ''}
+										${json.invalidText ? `invalidText='${json.invalidText}'` : ''} 
+										placeholder="${json.placeholder}"
+										size="${json.size}"
+										type='text' />`
+									: ''
+								}
+						</DatePicker>`;
+				}
+			},
+			v10: {
+				imports: ['DatePicker', 'DatePickerInput'],
+				code: ({ json }) => {
+					return `<DatePicker
+								${reactClassNamesFromComponentObj(json)}
```

Comment:
block indentation looks off

Reply:
same thing, in local, it looks fine

![image](https://github.com/IBM/carbon-ui-builder/assets/28711466/8cf5bb15-4f73-4c81-bd0b-534622c6e039)